File: Python_Workspace/Python_Tasks/16_05/3.py
#Abundant number
a=int(input("Enter the number:"))
sum=0
for i in range (1,a):
  if a%i==0:
    sum=sum+i
if sum>a:
  print(a,"is a Abundant number")
else:
  print(a,"is not a Abundant number")

File: C_Workspace/C_CheckList/Functions/3.c
//C program to find Sum of all Array Elements by passing array as an argument using User Define Functions.
#include <stdio.h>

int array(int [],int);

int array(int p[],int q)
{
    int c=0;
    printf("Sum of all Array Elements are:");
    for(int i=0;i<q;i++)
    {
        c=c+p[i];
    }
    printf(" %d",c);
}

int main()
{
    int n;
    printf("Enter the range:");
    scanf("%d",&n);
    int a[n];
    printf("Enter the numbers:");
    for(int i=0;i<n;i++)
    {
        scanf("%d",&a[i]);
    }
    array(a,n);
    return 0;
}
